heron/schedulers/src/java/com/twitter/heron/scheduler/slurm/SlurmController.java

```diff
@@ -140,7 +137,7 @@ public boolean killJob(String jobIdFile) {
     List<String> jobIdFileContent = readFromFile(jobIdFile);
     if (jobIdFileContent.size() > 0) {
       String[] slurmCmd = new String[]{"scancel", jobIdFileContent.get(0)};
-      return runProcess(null, slurmCmd, new StringBuilder(), new StringBuilder());
+      return runProcess(null, slurmCmd, new StringBuilder());
```

Comment:
adding another signature actually means we will add three or four more signatures... https://github.com/twitter/heron/blob/master/heron/spi/src/java/com/twitter/heron/spi/utils/ShellUtils.java#L63